helpdesk/templates/helpdesk/public_homepage.html

```diff
@@ -75,18 +75,14 @@ <h2 name='submit'>{% trans "Submit a Ticket" %}</h2>
 <form method='get' action="{% url 'helpdesk:public_view' %}">
 <fieldset>
     <div class="form-group {% if field.errors %}has-error{% endif %}">
-        <label for='id_ticket'>{% trans "Ticket" %}</label>
+        <label for='id_ticket'>{% trans "Ticket number" %}</label>
```

Comment:
Nitpick: s/number/Number/ ?  I need to see this, so maybe it makes visual sense.